### PR TITLE
docs(View): add documentation in `View`

### PR DESCRIPTION
**Pull request by jillney**
**Summary**
Changes:
- Add documentation in `View`.
- Add docs script in package.json
- Add `docz`, `docz-theme-default`,
  `react-art`, `react-native-web`
  and `react-router` to devDependencies.
- Add `.docz` in `.gitignore`
- Add `docrz` (i.e. docz set up file)
- Update `View` tests and snapshot tests (sidefix)

This PR fixes/implements the following **bugs/features**
* [ ] Add documentation in `View` component
* [ ] Update `View` tests

**Test plan (required)** for Documentation
1. Run `yarn docs:dev`
2. Go to http://127.0.0.1:3000/View

**Test plan (required)** for Testing
1. Run `yarn test`

**Closing issues**
CareLuLu/react-native-web-ui-components#44

### CODE REVIEW

File: docs/view.mdx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+<Playground>
+      <ThemeProvider>
+            <View class="main_container" style={{backgroundColor: '#20B2AA'}}>
+                  <Text>Text inside a View</Text>
+                  <View class="inner_container" style={{backgroundColor: '#FFB6C1'}}>
+                        <Text>Text inside another View.</Text>
+                  </View>
+            </View>
+      </ThemeProvider>
+</Playground>
```
Comment (CareLuLu-Gabriel) [inline docs/view.mdx]: Please use 2 spaces for indentation.

File: docs/view.mdx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+To following example changes the `View` background color using the `onRef` property.
+The `onRef` property is set to `{component => _view = component}`. The variable namely 
+`_view` is then used to set the native property of the component and change the
+background color.
+
+<Playground>
+  {() => {
+      isColorPink = true;
+      changeBackgroundColor = () => {
+            if(isColorPink){
```
Comment (CareLuLu-Gabriel) [inline docs/view.mdx]: Please use spaces between `if` and `(` and `)` and `{`.

File: docs/view.mdx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                  <View onRef={component => _view = component} style={{backgroundColor: '#FFB6C1'}}>
+                        <Text>Hello World!</Text>
+                        <Button 
+                              type='navy' 
+                              onPress={changeBackgroundColor}>Change Background Color
+                        </Button>
+                  </View>
+            </ThemeProvider>
+      );
+  }}
```
Comment (CareLuLu-Gabriel) [inline docs/view.mdx]: I think `View` no longer has `onRef`. You can remove the onRef section.

Comment (CareLuLu-Gabriel) [reply docs/view.mdx]: A note on the code, please use `let` and `const` to declare variables.

**Comment by jillney** [comment]
Hi Gabriel, Good day! The new PR fixes the following:
* Indentation
* Spacing
* Variable Declaration

The master branch still has `onRef` in `View` component. Hence, the documentation and tests for that are not deleted.

PR Link: https://github.com/CareLuLu/react-native-web-ui-components/pull/63

Thanks!

**Pull request closed without merging**